### PR TITLE
Renamed methods

### PR DESCRIPTION
**Pull request by petromw**

### CODE REVIEW

**Comment by petromw** [comment]
I changed the names of the test methods to be more aligned with the 'given-when-then' syntax

**Comment by petromw** [comment]
I added try catch blocks that might be helpful in the future when you are not using mock data anymore.

**Comment by petromw** [comment]
IntelliJ says it's not recommended to use the @Autowired tag there, but I read the documentation and I honestly can't decide if I agree. Here's the documentation if you want to puzzle it out https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.spring-beans-and-dependency-injection

File: src/main/java/com/enterprise/redcord/dao/MessageDAOStub.java [hunk 1]
```diff
@@ -25,7 +25,11 @@ public Message saveMessage(Message messageEntry) {
             latestMessageId = lastMessage.getMessageId();
         }
         messageEntry.setMessageId(latestMessageId + 1);
-        allMessages.put(messageEntry.getMessageId(), messageEntry);
+        try {
+            allMessages.put(messageEntry.getMessageId(), messageEntry);
+        } catch(Exception e) {
+            e.printStackTrace();
```
Comment (discospiff) [inline src/main/java/com/enterprise/redcord/dao/MessageDAOStub.java:31]: allMessages is simply a HashMap.  The put () method won't throw an exception.  So, this is not needed.

File: src/main/java/com/enterprise/redcord/dao/TopicDAOStub.java [hunk 2]
```diff
@@ -15,7 +15,12 @@
 
     @Override
     public Topic save(Topic topic) throws Exception {
-        allTopics.put(topic.getTopicId(), topic);
+        try {
+            allTopics.put(topic.getTopicId(), topic);
+        } catch(Exception e){
+            e.printStackTrace();
+        }
```
Comment (discospiff) [inline src/main/java/com/enterprise/redcord/dao/TopicDAOStub.java:22]: Ditto above.  You don't need to wrap a try-catch around a HashMap method.

File: src/test/java/com/enterprise/redcord/RedCordApplicationTests.java [hunk 3]
```diff
@@ -59,12 +59,12 @@ private void thenCreateNewMessageRecordAndReturnIt() throws Exception {
 
 
     @Test
-    void save_fetch() throws Exception  {
-        saveTopic();
-        assertSaved();
+    void saveTopicThenValidate() throws Exception  {
+        whenSaveTopic();
+        thenAssertSaved();
     }
```
Comment (discospiff) [inline src/test/java/com/enterprise/redcord/RedCordApplicationTests.java:65]: Good, though add givenXxx as well.

**Comment by discospiff** [comment]
The catch blocks aren't adding any value, so I would not implement them.

It's a good idea to use Given When Then syntax, though I recommend implementing it fully.

**Comment by dmaddx** [comment]
Try catches not needed for hash map. Will fully implement test method.

**Pull request closed without merging**